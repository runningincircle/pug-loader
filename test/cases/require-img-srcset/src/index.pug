p Hello World!

img(src=require('./image.png')).a
img(srcset=`${require('./image1.png')} 80w, ${require('./image2.png')} 90w` src=require('./image.png')).b