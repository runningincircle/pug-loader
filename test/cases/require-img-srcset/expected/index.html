<p>Hello World!</p><img class="a" src="/assets/images/image.697ef306.png"><img class="b" srcset="/assets/images/image1.697ef306.png 80w, /assets/images/image2.697ef306.png 90w" src="/assets/images/image.697ef306.png">